modules/cudf/src/types/mappings.ts

```diff
@@ -44,6 +44,12 @@ export enum Interpolation
   nearest    ///< i or j, whichever is nearest
 }
 
+export enum NullEquality
+{
+  equal,    // nulls compare equal
+  unequal,  // nulls compare unequal
+}
```

Comment:
FWIW in the joins PR I was just going to use a `nullEquality` boolean in places, since there are only two values and that seems less of a burden on users. But mainly we should just be consistent either way. @trxcllnt do you have a preference?

Reply:
:+1:. Unless Paul disagrees, I'm fine with switching to a boolean here.

Reply:
I think boolean here is fine! We use enums in C++ because calling a method with `cudf::blah(true, false, true)` is super weird, but we're using bag-o-options in JS, so it's obvious from the key name what the boolean means.

Reply:
Though if we're not using option-bag arguments, then the enum is probably the way to go (or `keyof` an enum like we do in [other places](https://github.com/rapidsai/node-rapids/blob/017cfb718298aead959f8e6cfbeb02c786af051b/modules/cudf/src/series/numeric.ts#L1401-L1406)).

Reply:
updated in 8d19015